.github/workflows/oclif-release.yml

```diff
@@ -31,7 +31,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: |
-          npm install
+          make install
+          npm list @twilio/cli-core
```

Comment:
we can remove this, but I feel this would help us in verifying the twilio-cli-core version for future runs.